hdav/hdav.py

```diff
@@ -128,14 +170,22 @@ def __init__(self, layers, *args, **kwargs):
         self.cursor_sagittal_y = pg.InfiniteLine(angle=0, pen=y_pen)
         self.cursor_sagittal_z = pg.InfiniteLine(angle=90, pen=z_pen)
 
-        self.cursor_pos = np.array(layers[0]['data'].shape) / 2
+        self.cursor_pos = np.array(self.shape) / 2
         self.draw()
 
+    def update_cursor(self, old_shape, new_shape):
+        self.set_clipped_cursor(
+            [new_shape[dim] * value / old_shape[dim] for dim, value in enumerate(self.cursor_pos)])
```

Comment:
Could this be simplified to just `np.array(new_shape) * value / np.array(old_shape)`?